setup.py

```diff
@@ -31,7 +31,7 @@ def run(self):
 
 setup(
     name='AxonDeepSeg',
-    python_requires='>=3.6, <3.7',
```

Comment:
Why not simply =3.7?

Reply:
I believe it's to allow Python versions from 3.7.0 to 3.7.7

Reply:
ah? i thought 3.7 do include all these? (ie when you install python, it fetches the latest patch)

Reply:
good point, thanks, i'll check as soon as i can (my workstation is down at the moment)

Reply:
The interval is required to allow versions from 3.7.0 to 3.7.7.
'3.7' and '=3.7' are invalid version specifier and '==3.7' returns `ERROR: Package 'AxonDeepSeg' requires a different Python: 3.7.7 not in '==3.7'`

Reply:
so i guess the solution is `>=3.7` as discussed here https://github.com/neuropoly/axondeepseg/issues/306

Reply:
Some dependencies are not compatible with 3.8, so we keep the interval `>=3.7, <3.8`.